sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestSuites.java

```diff
@@ -43,6 +43,7 @@
    * */
   public TestSuites() {
     // Empty constructor is required by xstream
+    this.testSuites = new ArrayList<>();
```

Comment:
@kinow I looks like constructor is not empty anymore:) It's worthy to remove comment above